Mathlib/Geometry/Manifold/AnalyticManifold.lean

```diff
@@ -0,0 +1,192 @@
+/-
+Copyright (c) 2024 Geoffrey Irving. All rights reserved.
```

Comment:
If I understand correctly, mathlib practice is to put the original author(s) of the code you're moving: in this case, using git blame to find out who wrote this code. Give a moment...

Reply:
Turns out this was written in mathlib4, in #6386. Please update the header accordingly.

Reply:
Oops, fixed. Apologies: I'd copied those lines from the broader commit, where most of the new file was mine.